site/content/operational_guide/resource_limits.md

```diff
@@ -53,6 +53,16 @@ per second safely with your deployment and you want to use the default lookback
 of `15s` then you would multiply 10,000 by 15 to get 150,000 as a max value with 
 a 15s lookback.
 
+The third limit `maxRecentlyQueriedSeriesDiskRead` caps the bytes associated with 
```

Comment:
Should we make each of the limits a subsection so it's clear what each section pertains to?